lib/overcommit/hook/pre_commit/es_lint.rb

```diff
@@ -8,9 +8,22 @@ module Overcommit::Hook::PreCommit
   # somewhere else. Example:
   #
   #   EsLint:
+  #     required_executable: 'npm'
   #     enabled: true
   #     command: ['npm', 'run', 'lint']
   #
+  # And currently this hook supports only eslint compact format, you can configure
+  # this in package.json. Example:
```

Comment:
Yes, that's fine. It also doesn't conflict with the advice above. You could either set `"lint": "eslint -f compact"` in `package.json`, or just use `"lint": "eslint"` and then set `command: ['npm', 'run', 'lint', '--', '-f', 'compact']` in `.overcommit.yml`.